### DISCUSSION TITLE
Upgrade to .NET 8.0 ?

### DISCUSSION THREAD

**Discussion by Osamious** [seerge/g-helper#2715] [General]
Great app! however I can't open it anymore because it says that I need to download .NET 7.0.19 which is out of support. Would you consider upgrading you program to .NET 8.0?
Thanks

<img width="299" alt="1" src="https://github.com/seerge/g-helper/assets/155423759/bfe22e59-7a5b-4d0e-84d1-1328f6d00a91">

**Comment by seerge** [comment]
@Osamious hello. Please make yourself familiar [with readme](https://github.com/seerge/g-helper/wiki/Requirements#requirements-mandatory). You just need to download .NET 7 to use the app, which is available and works fine.

Thanks

**Reply by Osamious** [reply]
Thanks for the reply, please note I have already gone through them and downloaded the mandatory files, but the error still persists.

**Reply by seerge** [reply]
You need to download and instal it
https://download.visualstudio.microsoft.com/download/pr/8091a826-e1c4-424a-b17b-5c10776cd3de/8957f21a279322d8fac9b542c6aba12e/dotnet-sdk-7.0.408-win-x64.exe

**Reply by Osamious** [reply]
![image](https://github.com/seerge/g-helper/assets/155423759/0fc83d2a-36d3-4044-971c-66e487a82c03)

Yes I ran the installers

**Reply by seerge** [reply]
@Osamious try to uninstall and install again. It's something broken or not working as it should in your specific system.

**Reply by Osamious** [reply]
@seerge tried reinstalling GHelper many times, wish I knew how to fix it :(

**Reply by seerge** [reply]
@Osamious you need to uninstall and reinstall .NET 7. G-Helper doesn't have installation at all, it's just a portable exe.

**Reply by Osamious** [reply]
![image](https://github.com/seerge/g-helper/assets/155423759/9774f7ab-fe04-4150-a408-c37c0b88032e)
![image](https://github.com/seerge/g-helper/assets/155423759/eca3c913-c14f-43f3-b31b-929734d208bb)

Done that too!

**Comment by seerge** [comment]
@Osamious try to install this one as well
https://download.visualstudio.microsoft.com/download/pr/08bbfe8f-812d-479f-803b-23ea0bffce47/c320e4b037f3e92ab7ea92c3d7ea3ca1/windowsdesktop-runtime-7.0.20-win-x64.exe

It's a .NET runtime from https://dotnet.microsoft.com/en-us/download/dotnet/7.0

**Reply by Osamious** [reply]
![image](https://github.com/seerge/g-helper/assets/155423759/52ac5e42-9466-46eb-b105-ab451c8502e7)
Legend, worked like a charm !

**Reply by seerge** [reply]
Congratulations
